modules/management/kubectl-apply/main.tf

```diff
@@ -87,15 +86,19 @@ module "kubectl_apply_manifests" {
 }
 
 module "install_kueue" {
-  source            = "./kubectl"
-  source_path       = local.install_kueue ? local.kueue_install_source : null
-  server_side_apply = true
-  wait_for_rollout  = true
-  depends_on        = [var.gke_cluster_exists]
+  source           = "./helm_install"
+  count            = var.kueue.install ? 1 : 0
+  release_name     = "kueue"
+  chart_repository = "oci://registry.k8s.io/kueue/charts"
```

Comment:
Is this the right chart repo ? As per their doc https://kueue.sigs.k8s.io/docs/installation/#install-by-helm it seems `oci://registry.k8s.io/kueue/charts/kueue`

Please test this PR for few blueprints along with job scheduling to confirm kueue installation is successful